app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def order_link(heading, content_item_attribute, order)
+    link_to heading, organisation_content_items_path(
```

Comment:
Thank you @selfthinker, I am merging this PR now, but we are more than happy to address this changes in future PRs.